pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
     "topojson>=1.5",
     "tqdm>=4.64.1",
     "s2>=0.1.9",
+    "pytorch-lightning>=1.0.0",
```

Comment:
I'm wondering if we should also have an explicit dependency on pytorch. You are, and we will be, importing it directly so I think this makes sense, but maybe we should discuss it a bit more first.

Reply:
Ok, I will add torch dependency